packages/openneuro-app/src/scripts/pet/dummy.jsx

```diff
@@ -25,7 +25,9 @@ class PETDummy extends React.Component {
                 <h1 className="pet-header">Coming soon!</h1>
                 <h2>
                   A section of OpenNeuro dedicated entirely to PET scans is
-                  currently under development.{' '}
+                  currently under development.</div>
+                  This section is co-developped with <a href="https://openneuropet.github.io/"> Open NeuroPET </a> 
+                  a project dedicated in sharing and curating brain PET imging data. {' '}
```

Comment:
Guessing the `</div>` is supposed to be a line break?

```suggestion
                  currently under development.<br />
                  This section is co-developed with <a href="https://openneuropet.github.io/">OpenNeuroPET</a>,
                  a project dedicated to sharing and curating brain PET imging data. {' '}
```

Reply:
depends the render you want </br> probably better (haven't tested, just added directly) - thx